ReIndex/index.js

```diff
@@ -8,14 +8,9 @@ async function azureIndexersSearchRequest(indexerName, method, body) {
   return azureSearchRequest(`indexers/${indexerName}`, method, body);
 }
 
-async function copyIndexDefinition(sourceIndexName, targetIndexName) {
-  const sourceIndexResponse = await azureIndexesSearchRequest(sourceIndexName, 'get');
-  if (sourceIndexResponse.statusCode === 200) {
-    const sourceIndexDefinition = sourceIndexResponse.body;
-    delete sourceIndexDefinition.name;
-    await azureIndexesSearchRequest(targetIndexName, 'delete');
-    await azureIndexesSearchRequest(targetIndexName, 'put', JSON.stringify(sourceIndexDefinition));
-  }
+async function recreateIndex(indexName, indexDefinition) {
+  await azureIndexesSearchRequest(indexName, 'delete');
```

Comment:
Should or can the statusCode of the response be checked to ensure it is ok rather than just assuming it was ok?